### PR TITLE
Account settings/Trusted Parties  (batch of fixes  1)

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[QA - Design] Account settings - Section and panel headings are the wrong size](https://trello.com/c/1V7E5s94/8033-qa-design-account-settings-section-and-panel-headings-are-the-wrong-size)

**Comment by leomendoza123** [comment]
[[QA] Remove duplicate record panel collapses when error messages should be displayed](https://trello.com/c/VH6QZv6P/8021-qa-remove-duplicate-record-panel-collapses-when-error-messages-should-be-displayed)

**Comment by leomendoza123** [comment]
[[QA] Trusted parties: no trusted individuals message](https://trello.com/c/fXeNyhww/8023-qa-trusted-parties-no-trusted-individuals-message)

**Comment by leomendoza123** [comment]
[[QA - Design] Account settings - Sign in form labels](https://trello.com/c/PMy0ssBb/8031-qa-design-account-settings-sign-in-form-labels)

**Comment by leomendoza123** [comment]
[[QA - Design] Account settings - Inline link styling](https://trello.com/c/VVXGSPp8/8030-qa-design-account-settings-inline-link-styling)

**Comment by leomendoza123** [comment]
[[QA - Design] Trusted parties - Type size and spacing](https://trello.com/c/8aARHRJI/8028-qa-design-trusted-parties-type-size-and-spacing)

**Comment by leomendoza123** [comment]
[[QA - Design] Trusted parties - Trusted individuals](https://trello.com/c/sZZr8n2g/8027-qa-design-trusted-parties-trusted-individuals)

**Comment by leomendoza123** [comment]
[[QA - Design] Trusted parties - Spacing tweaks on trusted orgs](https://trello.com/c/3NVG5BoD/8029-qa-design-trusted-parties-spacing-tweaks-on-trusted-orgs)

**Comment by leomendoza123** [comment]
[[QA] 'Name is private' tag not displayed in Users that trust you section](https://trello.com/c/avRHOfX0/8036-qa-name-is-private-tag-not-displayed-in-users-that-trust-you-section)

**Comment by leomendoza123** [comment]
[[QA - Design] Trusted parties - Add trusted individual modal](https://trello.com/c/ZYn0iQpV/8026-qa-design-trusted-parties-add-trusted-individual-modal)

**Comment by leomendoza123** [comment]
[[QA] Search for trusted individual, exact match is shown in add modal as well as results list](https://trello.com/c/O4oMfvCP/8035-qa-search-for-trusted-individual-exact-match-is-shown-in-add-modal-as-well-as-results-list)

File: src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html [hunk 1]
```diff
@@ -9,7 +9,13 @@
     class="mat-form-field-min no-hint"
     [ngClass]="{ mobile: isMobile, desktop: !isMobile }"
   >
-    <input matInput value="value" [(ngModel)]="searchValue" class="" />
+    <input
+      matInput
+      value="value"
+      [(ngModel)]="searchValue"
+      class=""
```
Comment (DanielPalafox) [inline src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html:16]: Could you remove this empty class @leomendoza123

**Review by amontenegro — APPROVED** [review]

File: src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html [hunk 2]
```diff
@@ -57,11 +66,12 @@
             >
 
             <a
+              class="underline"
```
Comment (DanielPalafox) [inline src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html:69]: attribute class its duplicated here

File: src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html [hunk 3]
```diff
@@ -106,11 +116,11 @@
             <mat-icon class="pointer" (click)="add(element)">add</mat-icon>
 
             <a
-              class="black-url"
+              class="black-url underline"
```
Comment (DanielPalafox) [inline src/app/account-trusted-parties/components/settings-trusted-individuals-search/settings-trusted-individuals-search.component.html:119]: black-url and underline have different color

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

**Pull request merged**